src/api/iframeFeatureEvent.api.js

```diff
@@ -43,3 +45,29 @@ export function sendFeatureInformationToIFrameParent(features) {
         // so let's not implement this format in the new viewer and see what happens.
     })
 }
+
+/**
+ * Is used to notify the parent the state of the app has changed. While embedding with VueJS, it's
+ * not possible to watch the iFrame src attribute, so an event is required to be notified of a
+ * children change.
+ *
+ * This is mainly used so that the iframe generator (menu share -> embed) can change the iframe
+ * snippet if the user decide to move / zoom the map while looking at the preview
+ */
+export function sendChangeEventToParent() {
+    if (!targetWindow) {
+        log.debug(
+            'Embed view loaded as root document of a browser tab, cannot communicate with opener/parent'
+        )
+        return
+    }
+    targetWindow.postMessage(
+        {
+            type: 'gaChange',
```

Comment:
We should start to document those API features that we implement in a wiki page !

Reply:
I've created a (self-assigned) ticket to do just that after my vacation
https://jira.swisstopo.ch/browse/PB-813